### PR TITLE
CAT-1700 Handle resume button correctly

### PR DESCRIPTION
**Pull request by Manu91**
I checked the required resources to only resume these resources and not all.

### CODE REVIEW

File: catroid/src/org/catrobat/catroid/stage/StageActivity.java [hunk 1]
```diff
@@ -323,4 +351,4 @@ public void run() {
 			}
 		});
 	}
-}
+}
```
Comment (fwinkelbauer) [inline catroid/src/org/catrobat/catroid/stage/StageActivity.java]: Please fix this by adding a new line

File: catroid/src/org/catrobat/catroid/stage/StageActivity.java [hunk 2]
```diff
@@ -189,16 +180,53 @@ public void pause() {
...
+			FaceDetectionHandler.resumeFaceDetection();
+		}
+
+		if ((requiredResources & Brick.BLUETOOTH_LEGO_NXT) > 0
+				|| (requiredResources & Brick.BLUETOOTH_PHIRO) > 0
+				|| (requiredResources & Brick.BLUETOOTH_SENSORS_ARDUINO) > 0) {
+			ServiceProvider.getService(CatroidService.BLUETOOTH_DEVICE_SERVICE).start();
+		}
+
+		if ((requiredResources & Brick.CAMERA_BACK) > 0) {
```
Comment (fwinkelbauer) [inline catroid/src/org/catrobat/catroid/stage/StageActivity.java]: could you merge this line with line 220 like you did in line 210?

`if ((requiredResources & Brick.CAMERA_BACK) > 0 || (requiredResources & Brick.CAMERA_FRONT) > 0)`

File: catroid/src/org/catrobat/catroid/stage/StageActivity.java [hunk 3]
```diff
@@ -189,16 +180,53 @@ public void pause() {
...
+		int requiredResources = ProjectManager.getInstance().getCurrentProject().getRequiredResources();
+		List<Sprite> spriteList = ProjectManager.getInstance().getCurrentProject().getSpriteList();
+
 		SensorHandler.startSensorListener(this);
-		FaceDetectionHandler.resumeFaceDetection();
 
-		CameraManager.getInstance().resumePreviewAsync();
+		for (Sprite sprite : spriteList) {
+			if (sprite.getPlaySoundBricks().size() > 0) {
+				stageAudioFocus.requestAudioFocus();
```
Comment (fwinkelbauer) [inline catroid/src/org/catrobat/catroid/stage/StageActivity.java]: Can we break the loop if this gets executed once?

**Comment by Manu91** [comment]
@fwinkelbauer done

**Pull request merged**